dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/validation/DefaultValidationService.java

```diff
@@ -49,24 +49,24 @@
 {
 
     @Qualifier( "org.hisp.dhis.tracker.validation.DefaultValidators" )
-    private final Validators validators;
+    private final Validator<TrackerBundle> validator;
```

Comment:
I think the qualifiers are wrong.
org.hisp.dhis.tracker.validation.validator.DefaultValidator should be on validator
and org.hisp.dhis.tracker.validation.validator.RuleEngineValidator should be on ruleEngineValidator

Reply:
Thank you for spotting this 😂